filters.py

```diff
@@ -0,0 +1,154 @@
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from .. import loader, utils
+
+import logging
+logger = logging.getLogger("FilterModule")
+
+
+def register(cb):
+    cb(Filters())
+
+
+class Filters(loader.Module):
+    """When you filter a text, it auto responds to it if a user triggers the word)"""
+
+    def __init__(self):
+        self.name = _("Filters")
+        self._me = None
+        self._ratelimit = []
+
+    async def client_ready(self, client, db):
+        self._db = db
+        self._client = client
+        self._me = await client.get_me()
+
+    async def filtercmd(self, message):
+        """Adds a filter into the list."""
+        args = (utils.get_args_raw(message)).strip()
+        reply = await message.get_reply_message()
+        chatid = str(message.chat_id)
+        filters = self._db.get("FilterModule", "filters")
+        if not args:
+            await message.edit("<b>Enter a name for the filter first!</b>")
+            return
+        else:
+            args = args.split(" ")
+            name = args[0]
+        if not filters:
+            filters = {}
+            filters.setdefault(chatid, {})
+        if chatid not in filters:
+            filters.setdefault(chatid, {})
+        if not reply:
+            if len(args) == 1:
+                await message.edit("<b>Please reply to a message or enter a text to save as filter.!</b>")
+                return
+            else:
+                value = " ".join(args[1::])
+                msg_to_log = await self._db.store_asset(value)
+        else:
+            value = reply.text
+            msg_to_log = await self._db.store_asset(reply)
+        filters[chatid][name] = str(msg_to_log)
+        self._db.set("FilterModule", "filters", filters)
+        await message.edit("<b>Text ''{}'' has been successfully filtered in this chat.</b>".format(name))
+
+    async def stopcmd(self, message):
+        """Removes a filter from the list."""
+        filtern = str(utils.get_args_raw(message))
+        filters = self._db.get("FilterModule", "filters")
+        chatid = str(message.chat_id)
+        if not filtern:
+            await message.edit("<b>Please specify the name of the filter.</b>")
+            return
+        try:
+            del filters[chatid][filtern]
+            await message.edit("<b>Filter ''{}'' successfully removed from the chat.</b>".format(filtern))
+            self._db.set("FilterModule", "filters", filters)
+        except Exception:
```

Comment:
NO NO NO NO NO NEVER EVER DO THIS

1. You should be only catching the line that throws, i.e. `del filters[...`
2. You should use an `else` clause on the if to run code only if it worked
3. Remove the formatting from the strings
4. Make the strings translatable
5. NEVER `EXCEPT EXCEPTION`. You want `except KeyError`